tests/pipeline/test_extraction.py

```diff
@@ -327,9 +325,9 @@ def test_run_detection_sextractor( decam_example_reduced_image_ds ):
     # wrong.
 
     assert ds.sources.psffluxadu()[0].min() == 0.0
-    assert ds.sources.psffluxadu()[0].max() == pytest.approx( 1726249.0, rel=1e-5 )
-    assert ds.sources.psffluxadu()[0].mean() == pytest.approx( 48067.805, rel=1e-5 )
-    assert ds.sources.psffluxadu()[0].std() == pytest.approx( 169444.77, rel=1e-5 )
+    assert ds.sources.psffluxadu()[0].max() == pytest.approx( 1716295.2, rel=1e-5 )
+    assert ds.sources.psffluxadu()[0].mean() == pytest.approx( 48076.523, rel=1e-5 )
+    assert ds.sources.psffluxadu()[0].std() == pytest.approx( 169310.296, rel=1e-5 )
 
```

Comment:
The values I get when I run this locally are a little bit different (the change is not small, but certainly outside 1e-5). 
I don't know why there would be differences (maybe sextractor versions?). 
Maybe we should make the `rel` a little bigger so it includes these variations, otherwise it would be good to figure out what causes the difference.

Reply:
Weird... what version of sextractor are you running?  The one in the docker image is:

```
root@646f46408b26:/seechange# source-extractor --version
Source Extractor version 2.25.0 (2018-02-08)
```

I'm fine bumping the rel to 1e-4.  Bigger than that, I'd want to know why there were differences.

Reply:
Ah!  I bet I know the reason they were different -- it was the changes you made to psf_sampling.  When you revert that, these should go back to what they were before.